### ISSUE TITLE
Empty network request logs in new Firefox windows?

### ISSUE THREAD

**Issue by jwonggj** [uBlock-LLC/uBlock#1448]
uBlock v0.9.5.0
FIrefox 38.0.5
OS X 10.10.3

Steps:
1. Open a new browser window, set to show a blank page.
2. Navigate to "Go to Request Log" in the dropdown.
3. Empty request log as shown. Note the blank menu from the chart icon in the top right and the empty list of tabs in the dropdown on the top left (no "Behind the Scenes")
![emptylog](https://cloud.githubusercontent.com/assets/12869463/8143279/5a17de8e-11e7-11e5-9e8a-ef49fda23b45.png)
4. Load any website.
5. Visit the request log again, still empty.

I've been able to resolve this by simply disabling and re-enabling uBlock from the Add-on list, which thankfully does not require a restart. The bug happens as long as the window starts empty; if I start browsing and then check the log, it will still be empty until I disable/enable uBlock. The bug does not seem to happen when I set new windows to display something when starting.

**Comment by sander85** [comment]
I'm seeing the same thing and from Browser Console I can see this:

TypeError: win is null             (chrome://ublock/content/js/vapi-background.js:367:4)

Should be this function:

var getTabBrowser = function(win) {
    return vAPI.fennec && win.BrowserApp || win.gBrowser || null;
};

**Comment by AlexVallat** [comment]
Sorry it's taken me so long to reply to this, I've been away. I haven't been able to reproduce this yet, but I have some ideas that might narrow it down: I notice from your screenshot it's a private browsing window - are you seeing the issue with normal windows too? Is it only with a new window, or with a new tab too? Does it happen if you just start the browser from exited with a blank page, or only if it was already running and you open a new one?

Unfortunately, if this turns out to be OS-X specific then I will not be able to reproduce or likely fix it; I don't have access to that OS. If you can reproduce the issue on Windows or Linux that would be very helpful.

**Comment by jwonggj** [comment]
Hi Alex,

I’ve attempted the described tests. In retrospect my description of the bug was inaccurate.

The bug appears only when the very first normal window is closed before uBlock being activated. For instance if I start FF -> first window opens -> open new private window and  check the network log, it works fine. I then tried browsing around in both windows, uBlock continued to work fine. When the first (normal) window was closed, the network log went blank. From this test, it would seem that closing the first window is the trigger.

After the bug is triggered, no amount of opening or closing windows and tabs fixes it. uBlock has to be reset.

Prior to the bug triggering, opening new tabs and windows did not trigger the bug.

When I opened two normal windows and a private window, closing the original normal window was sufficient to trigger the bug.
